ecleankernel/file.py

```diff
@@ -91,6 +96,38 @@ def __init__(self,
         super().__init__(path, KernelFileType.KERNEL)
         self.internal_version = self.read_internal_version()
 
+    def decompress_raw(self) -> bytes:
+        f = open(self.path, 'rb')
+        magic_dict = {
+            b'\x1f\x8b\x08': 'gzip',
+            b'\x42\x5a\x68': 'bz2',
+            b'\xfd\x37\x7a\x58\x5a\x00': 'lzma',
+            b'\x5d\x00\x00': 'lzma',
+            b'\x04\x22\x4d\x18': 'lz4.frame',
+            b'\x28\xb5\x2f\xfd': 'zstandard',
+            b'\x89\x4c\x5a\x4f\x00\x0d\x0a\x1a\x0a': 'lzo'
```

Comment:
Nit because I'll probably miss it later. It's generally a good idea to add extra `,` to make smaller diffs when adding stuff.

```suggestion
            b'\x89\x4c\x5a\x4f\x00\x0d\x0a\x1a\x0a': 'lzo',
```